packages/vue-instantsearch/rollup.config.js

```diff
@@ -54,11 +53,6 @@ function outputs(vueVersion) {
     vuePlugin({ compileTemplate: true, css: false }),
     commonjs(),
     json(),
-    buble({
-      transforms: {
-        dangerousForOf: true,
-      },
-    }),
```

Comment:
It's unmaintained and not needed anymore

Reply:
we used buble back in the day as the output was smaller than babel. Fine to switch to a maintained alternative though

Reply:
With/without babel it's a 0.10kb difference only, I forgot to replace it with babel, just did !